client/my-sites/hosting/sftp-card/index.js

```diff
@@ -136,34 +135,42 @@ const SftpCard = ( {
 			<CardHeading>{ translate( 'SFTP Credentials' ) }</CardHeading>
 			<div className="sftp-card__body">
 				<p>
-					{ translate( "Access and edit your website's files directly by using an SFTP client." ) }
+					{ username
+						? translate(
+								'Use the credentials below to access and edit your website using an SFTP client. {{a}}Learn more about SFTP on WordPress.com{{/a}}.',
+								{
+									components: {
+										a: (
+											<ExternalLink
+												icon
+												target="_blank"
+												href={ localizeUrl( 'https://en.support.wordpress.com/sftp/' ) }
+											/>
+										),
+									},
+								}
+						  )
+						: translate(
+								"Access and edit your website's files directly by using an SFTP client."
+						  ) }
 				</p>
 			</div>
 			{ displayQuestionsAndButton && (
 				<div className="sftp-card__questions">
 					<Accordion title={ translate( 'What is SFTP?' ) }>
 						{ translate(
-							"{{p}}It's a way for you to access the files and folders on a website via a client program such as {{a}}Filezilla{{/a}} on your local computer.{{/p}}" +
-								'{{p}}SFTP stands for Secure File Transfer Protocol (or SSH File Transfer Protocol). It was designed as an extension of the SSH (Secure SHell) protocol. The “secure” part is because it is run over a secure channel, in this case SSH.{{/p}}' +
-								'{{p}}SFTP is not to be confused with File Transfer Protocol (FTP) which is similar but not secure. Thanks to the security provided by SFTP you can rest easy knowing that your files and your site are safe and sound.{{/p}}' +
-								'{{p}}A variety of custom plugins and themes may ask you to create specific folders or add files via SFTP however, generally speaking, SFTP is not required for your site to function.{{/p}}',
-							{
-								components: {
-									p: <p />,
-									a: <a href={ FILEZILLA_URL } />,
-								},
-							}
-						) }
-					</Accordion>
-					<Accordion title={ translate( 'How to use SFTP to access your site' ) }>
-						{ translate(
-							'{{p}}Click on "Enable SFTP Credentials". Once you’ve enabled SFTP, you’ll see your SFTP URL, Port Number, Username and Password. Keep these credentials safe!{{/p}}' +
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( 'Enable SFTP Credentials' )` *ES Score: 7*